src/index.html

```diff
@@ -11,8 +11,53 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
You should not link a SCSS file in your HTML. SCSS is a preprocessor language and browsers can't interpret it. You should compile your SCSS to CSS and then link the CSS file in your HTML.